netclient/daemon/systemd.go

```diff
@@ -80,25 +59,17 @@ WantedBy=timers.target
 			return err
 		}
 	}
-
-	if !ncutils.FileExists("/etc/systemd/system/netclient.timer") {
-		err = os.WriteFile("/etc/systemd/system/netclient.timer", timerbytes, 0644)
-		if err != nil {
-			log.Println(err)
-			return err
-		}
-	}
-
 	_, _ = ncutils.RunCmd("systemctl enable netclient.service", true)
 	_, _ = ncutils.RunCmd("systemctl daemon-reload", true)
-	_, _ = ncutils.RunCmd("systemctl enable netclient.timer", true)
-	_, _ = ncutils.RunCmd("systemctl start netclient.timer", true)
+	_, _ = ncutils.RunCmd("systemctl start netclient.server", true)
```

Comment:
I think this should be netclient.service?

Reply:
probably why it wasn't starting :1st_place_medal: